boards/arm/lpcxpresso55s36/doc/index.rst

```diff
@@ -67,6 +67,8 @@ The Zephyr configuration for the LPC55S36 board (lpcxpresso55s36) is as follows:
 +-----------+------------+-------------------------------------+
 | CAN       | on-chip    | canbus                              |
 +-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
```

Comment:
Can we add a line here similar to what is present in #49296, like the following:

| IAP | on-chip | flash programming
| --- | --- | --- |